### PR TITLE
Slider Working!

### LINKED ISSUE #79: add image carousel to info panel
maybe there is a way to select for images that match the sidebar orientation? ie wide vs tall images

@hollydo suggests slick slider library

sad current image for _Agonis flexuosa_, a species that would really benefit from this update:
Screen Shot 2019-08-20 at 4 05 38 PM

### PR DESCRIPTION
**Pull request by theswerd**
resolves #79

This is a slider to allow users to school through the images

Original:
Screen Shot 2019-09-10 at 6 14 45 PM
New:
Screen Shot 2019-09-10 at 6 14 58 PM

### CODE REVIEW

**Comment by emillipede** [comment]
thank you @theswerd this feature is great! I also appreciate how you documented the PR

do you know what causes the blue "selected" frame to appear around the arrows? I think it looks a little distracting.

Screen Shot 2019-09-11 at 2 14 43 PM
Screen Shot 2019-09-11 at 2 11 00 PM

when I resize the window, I also notice that the colors of the arrow and the "selected" frame change. please let me know if you think it's possible to remove the frame. 

Screen Shot 2019-09-11 at 2 17 46 PM

once that's sorted out, I think it'll be good to merge. I'd like to get Jess' input about how to size/weight the arrows, but I it's minor

thank you again!

File: public/css/main.css [hunk 1]
```diff
@@ -194,6 +218,32 @@ select.shadow {
...
+#rightButton button{
+  font-size: 60px;
+  font-weight: bolder;
+  color: white;
+  top: 50%;
+  transform: translateY(-50%);
+  background: transparent;
+}
+#rightButton:hover button{
+  color: blue;
```
Comment (emillipede) [inline public/css/main.css]: for me, it only seems to do the hover color change in when it's sized to a mobile/tablet. I recommend no hover, I think it's too busy

File: public/js/Sidebar.js [hunk 2]
```diff
@@ -103,7 +105,10 @@ a2a_config.templates = a2a_config.templates || {};
       this.image.style.backgroundImage = 'url(' + tree.images[1].url + ')';
       this.image.style.backgroundSize = 'cover';
       this.image.classList.remove('hidden');
+        
       this.imageCreditLink.href = tree.images[1].author.url;
+      console.log(tree.images.length);
```
Comment (emillipede) [inline public/js/Sidebar.js]: I'd remove this line

Comment (theswerd) [reply public/js/Sidebar.js]: @emillipede just made those changes

**Comment by emillipede** [comment]
awesome, thank you @theswerd !

**Pull request merged**